src/instructlab/model/chat.py

```diff
@@ -677,19 +671,14 @@ def chat_cli(
     qq,
     greedy_mode,
     max_tokens,
-    tls_insecure,
-    tls_client_cert: Optional[str] = None,
-    tls_client_key: Optional[str] = None,
-    tls_client_passwd: Optional[str] = None,
 ):
     """Starts a CLI-based chat with the server"""
-    cert = get_ssl_cert_config(tls_client_cert, tls_client_key, tls_client_passwd)
-    verify = not tls_insecure
+    logger = ctx.obj.logger
     client = OpenAI(
         base_url=api_base,
-        api_key=api_key,
+        api_key=ctx.params["api_key"],
```

Comment:
Where `ctx.params` coming from? Where is it initialized?

Reply:
It is coming from Click module.

Reply:
I suppose it is initialized [here](https://github.com/pallets/click/blob/923d197b56caa9ffea21edeef5baf1816585b099/src/click/core.py#L920)